pkg/modulewriter/tfwriter.go

```diff
@@ -64,9 +64,19 @@ func createBaseFile(path string) error {
 	return err
 }
 
+func handleBlueprintVariables(hclBytes []byte) []byte {
```

Comment:
Since they are in the same class, you could simply use `escapeBlueprintVariables` from hcl_utils.go in tfwriter. That would replace the calls to `handleBlueprintVariables()` with `escapeBlueprintVariables()` in the write* functions.

Eventually, the goal is to move much more from tfwriter and packerwriter into shared functions, this is a good opportunity to do that.

Reply:
Absolutely! I should have thought that before.